src/Error/FormattedError.php

```diff
@@ -216,6 +216,7 @@ static function (SourceLocation $loc): array {
             $formattedError = self::addDebugEntries($formattedError, $exception, $debug);
         }
 
+        $formattedError['code'] = $e->getCode();
```

Comment:
This is discouraged by the GraphQL specification, see http://spec.graphql.org/draft/#note-5c13b

Reply:
We should not pursue this, right? So it stays protocol agnostic.

Reply:
Well the `code` in this case is an internal PHP error code, not an HTTP status code. But as the spec says, if anywhere it should go under the `extensions` key.